doc/src/user_guide.rst

```diff
@@ -397,13 +403,13 @@ Parameterises the time-integration scheme used by the solver with
 
            *float*
 
-        - ``pseudo-dt`` --- pseudo time-step
```

Comment:
Not sure about these changes?

Reply:
What's the issue? User guide says `controller — pseudo time-step controller`

Reply:
I think we're just re-arranging two entries in the user guide, but am unsure why?  Or am I missing something?

Reply:
I did rearrange two entries to group the physical terms and pseudo terms separately, but the current guide incorrectly says the controller option is the _pseudo_ time-step controller: 
```
dt --- time-step
    float
pseudo-dt --- pseudo time-step
    float
controller --- pseudo time-step controller
    none
```

so I also changed the description of controller:
```
dt --- time-step
    float
controller --- time-step controller
    none
pseudo-dt --- pseudo time-step
    float
```

Reply:
Makes sense.  Can you break out into its own PR?